src/components/login/SignIn.tsx

```diff
@@ -68,9 +94,24 @@ export const SignIn = ({ changeMode }: SignInProps) => {
           <InputLeftElement pointerEvents="none">
             <AiFillLock />
           </InputLeftElement>
+          <InputRightElement
+            onClick={() => {
+              if (visible === 'password') {
+                setVisible('string');
+              } else {
+                setVisible('password');
+              }
```

Comment:
i didn't got why did you put these names `passowrd` and `string` on the setVisible state.

visible stands for binary things, or is visible or is not visible, so IMHO should be an boolean.

Reply:
I used these values ​​because I use this state variable for two different things, so I avoided creating one more unnecessarily